app/Models/Event.php

```diff
@@ -47,4 +54,32 @@ public function league()
     {
         return $this->belongsTo(\App\Models\League::class);
     }
+
+    public function getScheduledDatesAttribute()
+    {
+        $dates = $this->sessions()
+            ->orderBy('start_time')
+            ->pluck('start_time')
+            ->unique();
+
+        return [
+            $dates->first(),
+            $dates->last()
```

Comment:
Why are we only returning the first and last session start dates? Since the dates are unique is there any reason to not just return them all? Im sure there is some situation where the list of all start dates would be useful.

Reply:
its mostly for brevity, this effectively represents the date range for the event, we _could_ return the whole collection, sure, but that's unnecessary for the use case of this accessor

Reply:
okay sounds good. in that case i have nothing else to add or change